.gitignore

```diff
@@ -1,4 +1,14 @@
-# Based on https://github.com/github/gitignore/blob/master/Node.gitignore
+# Yarn https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
```

Comment:
This root `.gitignore` file was generated years ago and contained a lot of "extras" that don't apply to FEM. I've cleaned it up and included the recommended Yarn files.